packages/fractor-typoscript/src/Factory/PrettyPrinterFormatFactory.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace a9f\FractorTypoScript\Factory;
+
+use a9f\FractorTypoScript\Configuration\TypoScriptProcessorOption;
+use a9f\FractorTypoScript\ValueObject\TypoScriptPrettyPrinterFormatConfiguration;
+use Symfony\Component\DependencyInjection\ParameterBag\ContainerBagInterface;
+
+final readonly class PrettyPrinterFormatFactory
+{
+    public function __construct(
+        private ContainerBagInterface $parameterBag
+    ) {
+    }
+
+    public function create(): TypoScriptPrettyPrinterFormatConfiguration
```

Comment:
This should be part of the static factory method instead. No need for two indirections: a factory class that calls a static factory method inside the class.

Reply:
I looked at what rector did and implemented it the same way like there. Feel free to improve it!